superset/views/core.py

```diff
@@ -485,6 +485,7 @@ def annotation_json(  # pylint: disable=no-self-use
         self, layer_id: int
     ) -> FlaskResponse:
         form_data = get_form_data()[0]
+        force = request.args.get("force") == "true"
```

Comment:
I seem to recall there being a util for parsing truthy values, but I wasn’t able to find it anywhere, e.g. "true", "1", 1, "True" all evaluating to `True` etc. If someone knows where this is please let me know.

Reply:
Maybe it's here?
https://github.com/apache/superset/blob/5aef6bccd1844bf2f318fef3d7abaefbc326e78a/superset/views/core.py#L1023-L1036

```suggestion
        force = request.args.get("force", "").lower() == "true"
```

Reply:
I found this, so I added a util for it as there didn't seem to be one yet: https://docs.python.org/2/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool

Reply:
Thanks, Ville!